extensions/amp-analytics/0.1/requests.js

```diff
@@ -281,12 +302,26 @@ export class RequestHandler {
         'constructBatchSegments_ with invalid batchingPlugin function');
 
     return Promise.all(batchSegmentsPromise).then(batchSegments => {
+      const batchSegmentsStrs = {
+        requestUrl: '',
+      };
       try {
-        return this.batchingPlugin_(baseUrl, batchSegments);
+        if (!this.useBody_) {
+          batchSegmentsStrs.requestUrl =
```

Comment:
After thinking about this. We should be supporting the `batchingPlugin_` regardless of  `this.useBody_` value. And `batchingPlugin_` that support `useBody` should return a string, and a requestBody object. 
You don't need to make the change in this PR. I think it is OK to say that `batchingPlugin` and `useBody` doesn't work together at initial stage. With that, do you feel safe to revert the change here?